### PR TITLE
Feature login alert

### PR DESCRIPTION
**Pull request by NeHerkus**
Created a service and a component for showing alerts

### CODE REVIEW

File: src/app/services/alert.service.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        } else {
+          this.subject.next();
+        }
+      }
+    });
+  }
+
+  success(message: string, keepAfterNavigationChange = false) {
+    this.keepAfterNavigationChange = keepAfterNavigationChange;
+    this.subject.next({type: 'success', text: message});
```
Comment (gpiliponyte) [inline src/app/services/alert.service.ts]: better to have an interface for this object and replace the any type

Comment (gpiliponyte) [inline src/app/services/alert.service.ts]: types go inside an enum

File: src/app/services/alert.service.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import {Injectable} from '@angular/core';
+import {Router, NavigationStart} from '@angular/router';
+import {Observable, Subject} from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AlertService {
+  private subject = new Subject<any>();
```
Comment (gpiliponyte) [inline src/app/services/alert.service.ts]: maybe messageSource would be a better name

File: src/app/alert/alert.component.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import {AlertService} from '../services/alert.service';
+
+@Component({
+  selector: 'app-alert',
+  templateUrl: './alert.component.html'
+})
+
+export class AlertComponent implements OnInit, OnDestroy {
+  private subscription: Subscription;
+  message: any;
```
Comment (gpiliponyte) [inline src/app/alert/alert.component.ts]: type any is already implicitly applied when no value is assigned on definition (you should still create an interface for the expected object)

File: src/app/alert/alert.component.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<div *ngIf="message"
+     [ngClass]="{ 'alert': message,
+     'alert-success': message.type === 'success',
```
Comment (gpiliponyte) [inline src/app/alert/alert.component.html]: would be pretty if the types were in an enum

File: src/app/alert/alert.component.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+import {AlertService} from '../services/alert.service';
+
+@Component({
+  selector: 'app-alert',
+  templateUrl: './alert.component.html'
+})
+
+export class AlertComponent implements OnInit, OnDestroy {
+  private subscription: Subscription;
```
Comment (gpiliponyte) [inline src/app/alert/alert.component.ts]: kind of vague naming

File: src/app/services/alert.service.ts (hunk 1 above)
Comment (NeHerkus) [reply src/app/services/alert.service.ts]: Done

Comment (NeHerkus) [reply src/app/services/alert.service.ts]: Done

File: src/app/services/alert.service.ts (hunk 2 above)
Comment (NeHerkus) [reply src/app/services/alert.service.ts]: Done

File: src/app/alert/alert.component.ts (hunk 3 above)
Comment (NeHerkus) [reply src/app/alert/alert.component.ts]: Done

File: src/app/alert/alert.component.html (hunk 4 above)
Comment (NeHerkus) [reply src/app/alert/alert.component.html]: Done

File: src/app/alert/alert.component.ts (hunk 5 above)
Comment (NeHerkus) [reply src/app/alert/alert.component.ts]: Done

**Pull request merged**